InterceptedService support for the client

Hello. 
Is there any chance to add InterceptedService support for the client? I get the following errors while I try to use it:
```
error[E0277]: the trait bound `ResponseBody: Default` is not satisfied
let client = AppServiceClient::with_interceptor(
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `Default` is not implemented for `ResponseBody`
note: required by a bound in `common::app::app_service_client::AppServiceClient::<T>::with_interceptor` T::ResponseBody: Default,
required by this bound in `common::app::app_service_client::AppServiceClient::<T>::with_interceptor`


error[E0599]: the method `set_app_property` exists for struct `AppServiceClient<InterceptedService<tonic_web_wasm_client::Client, AddJwtInterceptor>>`, but its trait bounds were not satisfied

client.set_app_property(req).await?;
       ^^^^^^^^^^^^^^^^^ method cannot be called on `AppServiceClient<InterceptedService<tonic_web_wasm_client::Client, AddJwtInterceptor>>` due to unsatisfied trait bounds
pub struct InterceptedService<S, F> {
doesn't satisfy `<_ as Service<request::Request<http_body::combinators::box_body::UnsyncBoxBody<tonic::codegen::Bytes, tonic::Status>>>>::Response = response::Response<_>`
doesn't satisfy `_: GrpcService<http_body::combinators::box_body::UnsyncBoxBody<tonic::codegen::Bytes, tonic::Status>>`
doesn't satisfy `_: Service<request::Request<http_body::combinators::box_body::UnsyncBoxBody<tonic::codegen::Bytes, tonic::Status>>>`
note: the following trait bounds were not satisfied:
`InterceptedService<tonic_web_wasm_client::Client, AddJwtInterceptor>: GrpcService<http_body::combinators::box_body::UnsyncBoxBody<tonic::codegen::Bytes, tonic::Status>>`
`<InterceptedService<tonic_web_wasm_client::Client, AddJwtInterceptor> as Service<request::Request<http_body::combinators::box_body::UnsyncBoxBody<tonic::codegen::Bytes, tonic::Status>>>>::Response = response::Response<_>`
`InterceptedService<tonic_web_wasm_client::Client, AddJwtInterceptor>: Service<request::Request<http_body::combinators::box_body::UnsyncBoxBody<tonic::codegen::Bytes, tonic::Status>>>`
```

Is the missing `Default` trait for `ResponseBody` a problem?

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -12,22 +12,22 @@ keywords = ["grpc", "grpc-web", "tonic", "wasm"]
 edition = "2021"
 
 [dependencies]
-base64 = "0.20.0"
+base64 = "0.21.0"
 byteorder = "1.4.3"
-bytes = "1.3.0"
-futures-util = "0.3.25"
-http = "0.2.8"
+bytes = "1.4.0"
+futures-util = "0.3.26"
+http = "0.2.9"
 http-body = "0.4.5"
 httparse = "1.8.0"
-js-sys = "0.3.60"
+js-sys = "0.3.61"
 pin-project = "1.0.12"
 thiserror = "1.0.38"
 tonic = { version = "0.8.3", default-features = false }
 tower-service = "0.3.2"
-wasm-bindgen = "0.2.83"
-wasm-bindgen-futures = "0.4.33"
+wasm-bindgen = "0.2.84"
+wasm-bindgen-futures = "0.4.34"
 wasm-streams = "0.3.0"
-web-sys = { version = "0.3.60", features = [
+web-sys = { version = "0.3.61", features = [
     "Headers",
     "Request",
     "Response",

--- a/src/body_stream.rs
+++ b/src/body_stream.rs
@@ -4,7 +4,7 @@ use std::{
 };
 
 use bytes::Bytes;
-use futures_util::{Stream, TryStreamExt};
+use futures_util::{stream::empty, Stream, TryStreamExt};
 use http_body::Body;
 use js_sys::Uint8Array;
 use wasm_streams::readable::IntoStream;
@@ -27,6 +27,14 @@ impl BodyStream {
                 bytes_vec.into()
             })
             .map_err(Error::js_error);
+
+        Self {
+            body_stream: Box::pin(body_stream),
+        }
+    }
+
+    pub fn empty() -> Self {
+        let body_stream = empty();
 
         Self {
             body_stream: Box::pin(body_stream),

--- a/src/response_body.rs
+++ b/src/response_body.rs
@@ -4,6 +4,7 @@ use std::{
     task::{Context, Poll},
 };
 
+use base64::{prelude::BASE64_STANDARD, Engine};
 use byteorder::{BigEndian, ByteOrder};
 use bytes::{BufMut, Bytes, BytesMut};
 use futures_util::ready;
@@ -45,7 +46,9 @@ impl EncodedBytes {
         let index = self.max_decodable();
 
         if self.raw_buf.len() >= index {
-            let decoded = base64::decode(self.buf.split_to(index)).map(Bytes::from)?;
+            let decoded = BASE64_STANDARD
+                .decode(self.buf.split_to(index))
+                .map(Bytes::from)?;
             self.buf.put(decoded);
         }
 
@@ -331,6 +334,24 @@ impl Body for ResponseBody {
                 // If stream is finished but state machine is not done, return error
                 return Poll::Ready(Err(Error::MalformedResponse));
             }
+        }
+    }
+}
+
+impl Default for ResponseBody {
+    fn default() -> Self {
+        Self {
+            body_stream: BodyStream::empty(),
+            buf: EncodedBytes {
+                encoding: Encoding::None,
+                raw_buf: BytesMut::new(),
+                buf: BytesMut::new(),
+            },
+            incomplete_data: BytesMut::new(),
+            data: None,
+            trailer: None,
+            state: ReadState::Done,
+            finished_stream: true,
         }
     }
 }

